src/Components/AppBar.tsx

```diff
@@ -1,77 +1,52 @@
 import React from 'react';
-import styled from 'styled-components';
 import MuiAppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
 import AddIcon from '@material-ui/icons/Add';
 import Tooltip from '@material-ui/core/Tooltip';
-
-import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
-import { Fab } from '@material-ui/core';
+import Fab from '@material-ui/core/Fab';
 import AssistantIcon from '@material-ui/icons/Assistant';
+import styled from 'styled-components';
 import MoreButton from './MoreButton';
 import { ADD_LOCATION_FORM } from '../constants';
 
-const useStyles = makeStyles((theme: Theme) =>
-  createStyles({
-    text: {
-      padding: theme.spacing(2, 2, 0),
-    },
-    paper: {
-      paddingBottom: 50,
-    },
-    list: {
-      marginBottom: theme.spacing(2),
-    },
-    subheader: {
-      backgroundColor: theme.palette.background.paper,
-    },
-    appBar: {
-      top: 'auto',
-      bottom: 0,
-      height: '60px',
-    },
-    grow: {
-      flexGrow: 1,
-    },
-    fabButton: {
-      position: 'absolute',
-      zIndex: 1,
-      top: -30,
-      left: 0,
-      right: 0,
-      margin: '0 auto',
-    },
-    searchContainer: {
-      position: 'relative',
-    },
-  })
-);
```

Comment:
Reverted these changes since the `useStyles` code had some unused classes, and the ones that we needed didn't leverage the MUI theming, so went with the less verbose `styled` components.